nemo/deploy/triton_deployable.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
The code structure of this directory is too flat. There are components that belong to specific domains in the same logical level as core functionality that determines abstract interfaces and base classes. Please see how to organize domain specific logic into subdirectories that are logical mirror of the nemo.core and nemo.collections.* hierarchy.

Reply:
Thanks for the suggestion. Moved the llm specific query code in a nlp folder. The other files are generic enough to keep it where it is IMO.